index.js

```diff
@@ -5,9 +5,7 @@ const { join, dirname } = require('path')
 
 var counter = 0
 
-function id () {
-  return process.pid + '.' + counter++
-}
+const tmpFile = (path) => join(dirname(path), '.' + process.pid + '.' + counter++)
```

Comment:
Can you use a top level function, or just nothing?

Reply:
Done.